internal/quickfort/parse.lua

```diff
@@ -102,11 +102,11 @@ end
 
 local function parse_start(modeline, start_pos, filename, marker_values)
     local _, start_str_end, start_str =
-            string.find(modeline, '^%s+start(%b())', start_pos)
+            string.find(modeline, '^%s+start%s*(%b())', start_pos)
     if not start_str or #start_str == 0 then return false, start_pos end
     local _, _, startx, starty, start_comment =
             string.find(start_str,
-                        '^%(%s*(%d+)%s*[;, ]%s*(%d+)%s*[;, ]?%s*(.*)%)$')
+                        '^%(%s*(%d+)%s-[;, ]%s-(%d+)%s-[;, ]?%s*(.*)%)$')
```

Comment:
If I'm reading https://www.lua.org/pil/20.2.html correctly, it looks like this didn't change any functionality. Is this just for readability?

Reply:
Yes, this worked before, but I changed to the non-greedy matcher so it would be clearer (as much as regexes can be clearer) that the space in the `[;, ]` character class was a valid required character.

Reply:
Oh, I was just looking at the table, and missed the explanation of greediness below it - makes sense!